data/DataTrans.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+
+import pandas as pd
+import numpy as np
+import csv
+Max_Value=3000
+Min_Value=0
+
+def get_Zone(Z_csv):
+    zone=pd.read_csv(Z_csv)
```

Comment:
Just double check: how did you generate that zone.csv? Besides, state like PR (Puerto Rico) and some others are excluded? Can we add some comments like which states are excluded?

Reply:
I assigned a code for each state of the U.S. 

At present, it contains 49 states, AK are not included.